packages/dds/map/src/test/memory/directory.spec.ts

```diff
@@ -34,48 +34,48 @@ describe("SharedDirectory memory usage", () => {
     });
 
     benchmarkMemory(new class implements IMemoryTestObject {
-        title = "Create empty directory";
-        minSampleCount = 500;
+        public title = "Create empty directory";
```

Comment:
Looks like a lot of these memory test interfaces could stand to have a bunch of `readonly` sprinkled throughout, though obv. not for this PR.

Reply:
Probably wouldn't hurt to do this now, since they are test-only. I'll go ahead and just do it 🙂